client/app/views/index.js

```diff
@@ -1,57 +1,60 @@
 'use strict';
 
-var _ = require('lodash');
-var h = require('../helpers/helpers');
+var _ = require('../helpers/utils');
```

Comment:
@viniciusgama the idea is to isolate lodash (and any other dependencies) in the helpers. I'm still using lodash but it's referenced only once in the `utils` file.

This way I can test whoever depends on this utility functions (mocking/stubbing) and if in any point of time we decide lodash/jquery/moment are too much we can replace them.